src/main/java/com/esri/samples/map/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -0,0 +1,233 @@
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.layers.ArcGISMapImageLayer;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalFolder;
+import com.esri.arcgisruntime.portal.PortalItem;
+import com.esri.arcgisruntime.portal.PortalUserContent;
+import com.esri.arcgisruntime.security.AuthenticationManager;
+import com.esri.arcgisruntime.security.OAuthConfiguration;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Dialog;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.control.SelectionMode;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.util.StringConverter;
+
+public class CreateAndSaveMapController {
+
+
+  @FXML
+  private MapView mapView;
+  @FXML
+  private TextField title;
+  @FXML
+  private TextField tags;
+  @FXML
+  private TextArea description;
+  @FXML
+  private ComboBox<PortalFolder> folderList;
+  @FXML
+  private ListView<Basemap> basemapList;
+  @FXML
+  private ListView<Layer> layersList;
+  @FXML
+  private Button saveButton;
+
+  private ArcGISMap map;
+  private Portal portal;
+
+  @FXML
+  private void initialize() {
+
+    // set basemap options
+    basemapList.getItems().addAll(Basemap.createStreets(), Basemap.createImagery(), Basemap
+        .createTopographic(), Basemap.createOceans());
+
+    // update basemap when selection changes
+    basemapList.getSelectionModel().select(0);
+    basemapList.getSelectionModel().selectedItemProperty().addListener(o ->
+      map.setBasemap(basemapList.getSelectionModel().getSelectedItem())
+    );
+
+    basemapList.setCellFactory(c -> new BasemapCell());
+
+    // create and set a map with the first basemap option
+    map = new ArcGISMap(basemapList.getSelectionModel().getSelectedItem());
+    mapView.setMap(map);
+
+    // set operational layer options
+    String worldElevationService = "http://sampleserver6.arcgisonline.com/arcgis/rest/services/WorldTimeZones/MapServer";
+    ArcGISMapImageLayer worldElevation = new ArcGISMapImageLayer(worldElevationService);
+    worldElevation.loadAsync();
+
+    String worldCensusService = "http://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer";
+    ArcGISMapImageLayer worldCensus = new ArcGISMapImageLayer(worldCensusService);
+    worldCensus.loadAsync();
+
+    layersList.getItems().addAll(worldElevation, worldCensus);
+    layersList.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
```

Comment:
Lead me to believe that I can show more than one layer, however application only lets me select one at a time.